server/public/styles.css

```diff
@@ -131,6 +131,38 @@ a:hover {
   background-color: #1a1c20;
 }
 
+.scroller {
+  background-color: #32363c;
+  border-radius: 5px;
+  height: calc(100vh - 166px);
+  overflow-y: scroll;
+}
+
+.post {
+  background-color: #4e545e;
+  color: #eaecee;
+  width: 100%;
+  height: auto;
+  margin-bottom: 1rem;
+  border-radius: 5px;
+  transition: background-color 0.25s ease;
+}
+
+.post:hover {
+  background-color: #1a1c20;
+}
+
+.post-board-container {
+  position: relative;
+  width: 100%;
+  height: 0;
+  padding-top: 100%;
+}
```

Comment:
This feels hacky to me. What's going on here?

Reply:
https://tylerduprey-52451.medium.com/a-perfect-square-with-css-964499440998